packages/components/ng-uirouter-layout/src/index.js

```diff
@@ -69,10 +70,19 @@ angular
         }
 
         if (get(state, 'layout.name') === 'modal') {
+          let componentName = get(state, state.views.modal ? 'views.modal.component' : 'component');
```

Comment:
get(state, "views.modal.component", "component") ?

if you're calling state.views.modal explicitly you loose the benefits of using lodash "get"

Reply:
Indeed